docs/development.md

```diff
@@ -45,18 +45,17 @@ Contributed projects extend the functionality of Drupal to add new features or a
 
 ```bash
 # Navigate to the installation profile repository.
-cd web/profiles/contrib/yalesites_profile
+cd web/profiles/custom/yalesites_profile
 # Require the new module without downloading packages in this folder.
 lando composer require drupal/PROJECT --no-update
 # Return the project repo root.
 cd ../../../..
 # Add the module to the project repository.
-lando composer require drupal/PROJECT
-# Do not check in changes to the project's composer.json file.
+lando composer require drupal/yalesites_profile
```

Comment:
I think I fixed it. 'install' is not helpful as it will only install what already exists in the lock file. You would need to delete any local lock file if you had one. But the update seems to work.
`lando composer update @yalesites-org/yalesites_profile`